test/e2e/specs/fakerp/admin.go

```diff
@@ -103,4 +134,182 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]"
 		By("comparing the update blob before and after an update")
 		Expect(reflect.DeepEqual(before, after)).To(Equal(true))
 	})
+
+	It("should be able to configure azure file persistent volumes", func() {
+		const (
+			prefix = "azure-file"
+		)
+
+		var (
+			pvName string
+		)
+
+		pvName = fmt.Sprintf("%s-pv", prefix)
+		// need a way to lessen the chances of name collision in storage account names
+		// and also they must lowercase letters and numbers
+		clusterSHA := sha1.Sum([]byte(azCli.Config.ResourceGroup))
+		// storage account names are capped at 24 chars
+		accountName := fmt.Sprintf("azf%x", clusterSHA)[:24]
```

Comment:
reuse randomStorageAccountName() - move it to package randomstring and export it.